src/site/content/en/blog/bluetooth/index.md

```diff
@@ -0,0 +1,549 @@
+get an error later when trying to access them.
+
+```js
+navigator.bluetooth.requestDevice({
+  filters: [{
+    name: 'Francois robot'
+  }],
+  optionalServices: ['battery_service']
+})
+.then(device => { /* ... */ })
+.catch(error => { console.log(error); });
+```
+
+Finally, instead of `filters` you can use the `acceptAllDevices` key to show all
+nearby Bluetooth devices. You will also need to define the `optionalServices`
+key to be able to access some services. If you don't, you'll get an error later
+when trying to access them.
+
+```js
+navigator.bluetooth.requestDevice({
+  acceptAllDevices: true,
+  optionalServices: ['battery_service']
+})
+.then(device => { /* ... */ })
+.catch(error => { console.log(error); });
+```
+
+{% Aside 'caution' %}
+This may result in a bunch of unrelated devices being shown in the chooser and
+energy being wasted as there are no filters. Use it with caution.
+{% endAside %}
+
+### Connect to a Bluetooth Device
+
+So what do you do now that you have a `BluetoothDevice` returned from
+`navigator.bluetooth.requestDevice`'s Promise? Let's connect to the Bluetooth
+remote GATT Server which holds the service and characteristic definitions.
+
+```js
+navigator.bluetooth.requestDevice({ filters: [{ services: ['battery_service'] }] })
+.then(device => {
+  // Human-readable name of the device.
+  console.log(device.name);
+
+  // Attempts to connect to remote GATT Server.
+  return device.gatt.connect();
+})
+.then(server => { /* ... */ })
+.catch(error => { console.log(error); });
+```
+
+### Read a Bluetooth Characteristic
+
+Here we are connected to the GATT Server of the remote Bluetooth device. Now we
+want to get a Primary GATT Service and read a characteristic that belongs to
+this service. Let's try, for instance, to read the current charge level of the
+device's battery.
+
+In the example below, `battery_level` is the [standardized Battery Level
+Characteristic].
+
+
+```js
+navigator.bluetooth.requestDevice({ filters: [{ services: ['battery_service'] }] })
+.then(device => device.gatt.connect())
+.then(server => {
+  // Getting Battery Service...
+  return server.getPrimaryService('battery_service');
+})
+.then(service => {
+  // Getting Battery Level Characteristic...
+  return service.getCharacteristic('battery_level');
+})
+.then(characteristic => {
+  // Reading Battery Level...
+  return characteristic.readValue();
+})
+.then(value => {
+  console.log('Battery percentage is ' + value.getUint8(0));
+})
+.catch(error => { console.log(error); });
+```
+
+If you use a custom Bluetooth GATT characteristic, you may provide either the
+full Bluetooth UUID or a short 16- or 32-bit form to
+`service.getCharacteristic`.
+
+Note that you can also add a `characteristicvaluechanged` event listener on a
+characteristic to handle reading its value. Check out the [Read Characteristic
+Value Changed Sample] to see how to optionally handle upcoming GATT
+notifications as well.
+
+```js
+...
```

Comment:
Please do a search-and-replace for `...` (three periods) and replace it with `…` (the ellipsis character)

I know this sounds kinda nitpicky but we do it so that the idea is properly announced to screen readers

Reply:
Done. Thank you!